### PR TITLE
[close #305] Fix supported version in tikv br README

### LINKED ISSUE #305: br: fix supported version in tikv br README
### 1. Describe the bug

Supported version need update in tikv br README as it support v5.x.x in pr[#231](https://github.com/tikv/migration/pull/282)

### 5. What is your migration tool and TiKV version? (Required)

- TiKV BR:

### PR DESCRIPTION
**Pull request by haojinming**
Signed-off-by: haojinming <jinming.hao@pingcap.com>

### What problem does this PR solve?

Issue Number: close #305 

Problem Description: 
Supported version need update in tikv br README as it support v5.x.x in prhttps://github.com/tikv/migration/pull/282

### Code changes

- No code

### Check List for Tests

This PR has been tested by at least one of the following methods:
- No code

### Side effects

- No side effects

### Related changes

- No related changes

### To reviewers

Please follow these principles to check this pull requests:

- **Concentration**. One pull request should only do one thing. No matter how small it is, the change does exactly one thing and gets it right. Don't mix other changes into it.
- **Tests**. A pull request should be test covered, whether the tests are unit tests, integration tests, or end-to-end tests. Tests should be sufficient, correct and don't slow down the CI pipeline largely.
- **Functionality**. The pull request should implement what the author intends to do and fit well in the existing code base, resolve a real problem for TiDB/TiKV users. To get the author's intention and the pull request design, you could follow the discussions in the corresponding GitHub issue or [internal.tidb.io](https://internals.tidb.io) topic.
- **Style**. Code in the pull request should follow common programming style. *(For Go, we have style checkers in CI)*. However, sometimes the existing code is inconsistent with the style guide, you should maintain consistency with the existing code or file a new issue to fix the existing code style first.
- **Documentation**. If a pull request changes how users build, test, interact with, or release code, you must check whether it also updates the related documentation such as READMEs and any generated reference docs. Similarly, if a pull request deletes or deprecates code, you must check whether or not the corresponding documentation should also be deleted.
- **Performance**. If you find the pull request may affect performance, you could ask the author to provide a benchmark result.

*(The above text mainly refers to [TiDB Development Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/review-a-pr.html#checking-pull-requests). It's also highly recommended to read about [Writing code review comments](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/review-a-pr.html#writing-code-review-comments))*

### CODE REVIEW

**Review by zeminzhou — APPROVED** [review]
LGTM～

File: br/README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ bin/tikv-br restore raw \
 
 - Allocate sufficient resources for backup and restoration.
 
-- TiKV-BR only support raw data backup/restoration in TiKV cluster with version >= `6.1.0`. May support TiKV cluster with 5.x versions in the future.
+- TiKV-BR only support raw data backup/restoration in TiKV cluster with version >= `5.0.0`.
```
Comment (pingyu) [inline br/README.md:66]:
```suggestion
- TiKV-BR only supports raw data backup/restoration in TiKV cluster with version >= `5.0.0`.
```

**Review by pingyu — APPROVED** [review]
Rest LGTM.

File: br/README.md (hunk 1 above)
Comment (haojinming) [reply br/README.md]: Fixed, PTAL~ thanks.

**Pull request merged**